bundles/persistence/org.openhab.persistence.mysql/java/org/openhab/persistence/mysql/internal/MysqlPersistenceService.java

```diff
@@ -153,14 +156,42 @@ public void unsetPersistentStateRestorer(PersistentStateRestorer persistentState
 	public String getName() {
 		return "mysql";
 	}
+	
+	/**
+	 * 
+	 * @param i
+	 * @return
+	 */
+	private String getItemType(Item i) {
+		Item item = i;		
+		if(i instanceof GroupItem){
+			item = ((GroupItem) i).getBaseItem();
+			if(item == null){//if GroupItem:<ItemType> is not defined in *.items using StringType
+				logger.debug("mySQL: Did not find ItemType for {}, 'GroupItem:<ItemType>' is not defined in *.items File. Try to use first Child-Members ItemType now.", i.getName());
+				item = ((GroupItem) i).getMembers().get(0);
+				if(item == null){
+					logger.debug("mySQL: ItemType NOT found! Use STRINGITEM for {}", i.getName());
```

Comment:
hmm … in fact the missing BaseItem is the problem not the missing ItemType. What about something like :"Cannot detect ItemType because the GroupItems' base type isn't set …"